nupic/encoders/coordinate.py

```diff
@@ -96,6 +104,20 @@ def getScalars(self, inputData):
     """See `nupic.encoders.base.Encoder` for more information."""
     return numpy.array([0]*len(inputData))
 
+  
+  @cache(maxsize=100, typed=False)
```

Comment:
I'd like to change this value to `self.cacheSize` but can't access instance (or class, global maybe yes) members. Is it possoble to achieve without a new wrapper around this wrapper?